solutions/src/test/java/ru/sberbank/school/task11/StreamsTest.java

```diff
@@ -0,0 +1,71 @@
+package ru.sberbank.school.task11;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+public class StreamsTest {
+
+    @Test
+    public void testToMap() {
+        Map<String, String> expect = new HashMap<>();
+        expect.put("pandaKey", "pandaValue");
+        expect.put("murKey", "murValue");
+        Map<String, String> result = Streams.of(Arrays.asList("panda", "mur"))
+                .toMap(v1 -> v1+ "Key", v2 -> v2 + "Value");
```

Comment:
WhitespaceAround: &#39;+&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1231184991>*